src/hooks/useDatabases.ts

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react';
+import { Datasource } from 'data/CHDatasource';
+
+export default (datasource: Datasource): string[] => {
```

Comment:
nit: Let's return `readonly array` as you did in `useColumns`. 
A rule of thumb: use `readonly array` by default. for cases when Grafana API expects a mutable array, we can make a copy in place `[...readonlyArray]`